src/index.js

```diff
@@ -11,7 +11,7 @@ parser.yy = astBuilder;
 
 module.exports = function parse(sourceCode) {
 	try {
-		return parser.parse(sourceCode);
+		return JSON.parse(JSON.stringify(parser.parse(sourceCode)));
```

Comment:
I'm not sure, but it is needed from since I return tokens text  as an object containing text and substitution info, otherwise AVA keeps failing. It see two identical objects as different.

Maybe I introduced somewhere some object with a prototype other than `Object`.

This should be investigated further, and this line removed. I could impact performance...